apps/src/p5lab/spritelab/commands/variableCommands.js

```diff
@@ -0,0 +1,16 @@
+export const commands = {
+  hideVariable(nameArg) {
+    if (!nameArg) {
```

Comment:
I wonder if we have to worry about the `!nameArg` case now that the field is part of the block. I don't think variable fields have the ability to return something falsey. Probably doesn't hurt to leave it just in case though!